### DISCUSSION TITLE
Nan or Out of Range from redis-parser

### DISCUSSION THREAD

**Discussion by haxplorer** [outline/outline#2933] [Self Hosting] [answered]
While trying to start the server, after populating the .env file with REDIS, AWS S3 and Postgres credentials, I'm getting this error from redis-parser. 

![image](https://user-images.githubusercontent.com/37192/149269600-b5c99cdb-3fcc-45eb-934c-ca3d278d24dd.png)

**Comment by tommoor** [comment, marked as answer]
I've never seen this error – highly likely the issue is with your `REDIS_` configuration values

**Reply by haxplorer** [reply]
REDIS_URL=redis://default:password@host:port/outline

This is my REDIS param. As per my understanding, user:password@host:port is the format that gets handled by redis-parser. Unless it is assuming that everything that follows the first colon would be the port.

**Reply by tommoor** [reply]
There should be nothing after the port – redis does not have named databases

**Reply by haxplorer** [reply]
Thanks Tom. That was the mistake
